plugins/CoreHome/templates/_dataTableFooter.twig

```diff
@@ -16,7 +16,8 @@
 
         {% if properties.show_search %}
             <span class="dataTableSearchPattern">
-                <input type="text" class="searchInput" length="15" />
+		<label for="widgetSearch_{{ properties.report_id }}" style="display:none;"> {{ 'General_Search'|translate }} for {{ visualization.config.translations.label }}</label>
```

Comment:
you can remove the `for` word here as it's good enough to have `Search Websites` or `Search Operating Systems` rather than `Search for Operating Systems`